.github/cloud-samples-tools/cmd/main.go

```diff
@@ -48,21 +48,31 @@ func main() {
 		if configFile == "" {
 			log.Fatalln("❌ no config file specified\n", usage)
 		}
-
 		diffsFile := flag.Arg(2)
 		if diffsFile == "" {
 			log.Fatalln("❌ no diffs file specified\n", usage)
 		}
+		pathsFile := flag.Arg(3)
+		affectedCmd(configFile, diffsFile, pathsFile)
 
-		affectedCmd(configFile, diffsFile)
+	case "setups":
+		configFile := flag.Arg(1)
+		if configFile == "" {
+			log.Fatalln("❌ no config file specified\n", usage)
+		}
+		pathsFile := flag.Arg(2)
+		if pathsFile == "" {
+			log.Fatalln("❌ no paths file specified\n", usage)
+		}
+		setupsCmd(configFile, pathsFile)
 
 	default:
 		log.Fatalln("❌ unknown command: ", command, "\n", usage)
 	}
 }
 
 // affected command entry point to validate inputs.
-func affectedCmd(configFile string, diffsFile string) {
+func affectedCmd(configFile string, diffsFile string, pathsFile string) {
```

Comment:
nit: godoc style is to use the exact function name as the very first string in the comment.

Suggestion:

```go
// affectedCmd validates inputs passed into the system
```

Reply:
Good catch, that came from a refactor and forgot to update the godoc.